featherpad/CMakeLists.txt

```diff
@@ -152,9 +150,12 @@ if(HAIKU)
   install(FILES ${HELP_FILE} DESTINATION "${CMAKE_INSTALL_PREFIX}/data")
   install(FILES ${HELP_TR_FILES} DESTINATION "${CMAKE_INSTALL_PREFIX}/data")
 elseif(APPLE)
-  install(TARGETS featherpad RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX})
-  install(FILES ${HELP_FILE} DESTINATION "${CMAKE_INSTALL_PREFIX}/${PROJECT_NAME}.app/featherpad")
-  install(FILES ${HELP_TR_FILES} DESTINATION "${CMAKE_INSTALL_PREFIX}/${PROJECT_NAME}.app/featherpad")
+  SET_TARGET_PROPERTIES(featherpad PROPERTIES RESOURCE ${HELP_FILE})
+  SET_TARGET_PROPERTIES(featherpad PROPERTIES RESOURCE ${HELP_TR_FILES})
```

Comment:
Build fails with:
```
...
CMake Error at featherpad/CMakeLists.txt:161 (SET_TARGET_PROPERTIES):
  SET_TARGET_PROPERTIES called with incorrect number of arguments.
...
```
`${HELP_TR_FILES}` should be quoted:
```
SET_TARGET_PROPERTIES(featherpad PROPERTIES RESOURCE "${HELP_TR_FILES}")
```
Since `HELP_TR_FILES` expanded into `data/help_pt_BR data/help_ja`:
```
file(GLOB HELP_TR_FILES data/help_*)
```

Reply:
@sikmir 
Would you make a PR for it?

Reply:
> @sikmir
> Would you make a PR for it?

Yes, but I'm trying to figure out why help files installation doesn't work properly (as @shlyakpavel already noticed). I guess something still wrong with these 4 lines, even with quoting.

Reply:
> installation doesn't work properly (as @shlyakpavel already noticed).

That's unsolvable mystery :D

Reply:
Also, @sikmir do you build it on Intel or M1 Mac?

Reply:
> Also, @sikmir do you build it on Intel or M1 Mac?

I have only Intel Mac.